hack/apiserver.config

```diff
@@ -1,27 +1,27 @@
-/registry/actions/,tenant_begin,tenant_end
-/registry/certificatesigningrequests/,tenant_begin,tenant_end
-/registry/controllerrevisions/,tenant_begin,tenant_end
-/registry/cronjobs/,tenant_begin,tenant_end
-/registry/daemonsets/,tenant_begin,tenant_end
-/registry/deployments/,tenant_begin,tenant_end
-/registry/events/,tenant_begin,tenant_end
-/registry/horizontalpodautoscalers/,tenant_begin,tenant_end
-/registry/ingress/,tenant_begin,tenant_end
-/registry/jobs/,tenant_begin,tenant_end
-/registry/leases/,tenant_begin,tenant_end
-/registry/limitranges/,tenant_begin,tenant_end
-/registry/networkpolicies/,tenant_begin,tenant_end
-/registry/persistentvolumeclaims/,tenant_begin,tenant_end
-/registry/persistentvolumes/,tenant_begin,tenant_end
-/registry/poddisruptionbudgets/,tenant_begin,tenant_end
-/registry/podpresets/,tenant_begin,tenant_end
-/registry/pods/,tenant_begin,tenant_end
-/registry/podsecuritypolicies/,tenant_begin,tenant_end
-/registry/podsecuritypolicy/,tenant_begin,tenant_end
-/registry/podtemplates/,tenant_begin,tenant_end
-/registry/replicasets/,tenant_begin,tenant_end
-/registry/resourcequotas/,tenant_begin,tenant_end
-/registry/statefulsets/,tenant_begin,tenant_end
-/registry/storageclasses/,tenant_begin,tenant_end
-/registry/volumeattachments/,tenant_begin,tenant_end
+/registry/actions/,,
+/registry/certificatesigningrequests/,,
+/registry/controllerrevisions/,,
+/registry/cronjobs/,,
+/registry/daemonsets/,,
+/registry/deployments/,,
+/registry/events/,,
+/registry/horizontalpodautoscalers/,,
+/registry/ingress/,,
+/registry/jobs/,,
+/registry/leases/,,
+/registry/limitranges/,,
+/registry/networkpolicies/,,
+/registry/persistentvolumeclaims/,,
+/registry/persistentvolumes/,,
+/registry/poddisruptionbudgets/,,
+/registry/podpresets/,,
+/registry/pods/,,
+/registry/podsecuritypolicies/,,
+/registry/podsecuritypolicy/,,
+/registry/podtemplates/,,
+/registry/replicasets/,,
+/registry/resourcequotas/,,
+/registry/statefulsets/,,
+/registry/storageclasses/,,
```

Comment:
Please remove /registry/storageclasses/ since it does not have namespace or tenant data